ansible/group_vars/proxy-hosts.yml

```diff
@@ -12,16 +12,25 @@ nginx_proxy_worker_processes: 4
 nginx_proxy_buffers: 32 16k
 
 nginx_proxy_ssl: True
-nginx_proxy_ssl_certificate: /etc/nginx/ssl/idr-fullchain.pem
-nginx_proxy_ssl_certificate_key: /etc/nginx/ssl/idr-privkey.pem
+nginx_proxy_ssl_certificate: "{{ idr_default_nginx_proxy_ssl_certificate | default('/etc/nginx/ssl/idr-fullchain.pem') }}"
+nginx_proxy_ssl_certificate_key: "{{ idr_default_nginx_proxy_ssl_certificate_key | default('/etc/nginx/ssl/idr-privkey.pem') }}"
 nginx_proxy_http2: True
 
 
 ######################################################################
 # OMERO.web proxy
 
+nginx_proxy_log_format: main_timed_cache_upstream
+
 nginx_proxy_websockets_enable: True
 
+nginx_proxy_upstream_servers:
+- name: omeroreadonly
+  balance: ip_hash
```

Comment:
`balance: ip_hash`

I hadn't realized this was the strategy. So assuming we want to downtime an RO, we'd need to update the nginx config first (and likely haproxy?) and _then_ shutdown the OMERO, right?